Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -1313,6 +1365,34 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
     }
 }
 
+public class EphemeralMessageEditor {
+    private unowned let channelController: ChatChannelController
```

Comment:
Ideally, in the official API, this would use the channel updater and not the controller directly